hw/xbox/nv2a/nv2a_stubs.c

```diff
@@ -19,21 +19,19 @@
  * along with this program; if not, see <http://www.gnu.org/licenses/>.
  */
 
-#define DEFINE_STUB(name, region_id) \
-    uint64_t name ## _read(void *opaque, \
-                           hwaddr addr, \
-                           unsigned int size) \
-    { \
-        reg_log_read(region_id, addr, 0); \
-        return 0; \
-    } \
-    void name ## _write(void *opaque, \
-                        hwaddr addr, \
-                        uint64_t val, \
-                        unsigned int size) \
-    { \
-        reg_log_write(region_id, addr, val); \
-    } \
+#define DEFINE_STUB(name, region_id)                                   \
+    uint64_t name##_read(void *opaque, hwaddr addr, unsigned int size) \
```

Comment:
Should use the same name as `DEFINE_PROTO` for first argument (was `prefix`, is `n`).